versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/RocksDBStoreConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.store.rocksdb;
+
+import org.immutables.value.Value.Default;
+import org.immutables.value.Value.Immutable;
+
+@Immutable
+public abstract class RocksDBStoreConfig {
+
+  @Default
+  public String getRefTableName() {
```

Comment:
Can you change this to use the prefix as the mongo store does so it's all matched up now?

Reply:
Please see how Mongo does it: https://github.com/stevelorddremio/nessie/blob/main/versioned/tiered/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoDBStore.java#L145

Reply:
I was a bit lost about mapping ColumnFamilyHandles to ValueTypes. I updated the map to use the same pattern as MongoDBStore and updated MongoDBStoreConfig with the extra TablePrefix atttribute, but I don't think this is quite correct.
Also the Config object is an Immutable and it doesn't look like we build it correctly in the TestRocksDBStore. So probably need a chat in the morning to clear this up. I set this up in rocks-config branch instead.